### DISCUSSION TITLE
Anyone know if Yahoo! is open to increasing the API limits?

### DISCUSSION THREAD

**Discussion by dvdrplus** [mattdodge/yahoofantasy#42] [General]
The API cuts out pretty quick if load on my app is high.

I see Yahoo! finance has a limit of like 10,000 per hour or something. The Yahoo! Sports is much much lower.

I can't find any support at all from Yahoo! regarding this. Does anyone have any experience here, or someone/where I could contact to get support from Yahoo! about this?

**Comment by mattdodge** [comment]
I'm unaware of any way to request an increase in API limits. This repo has some [rate throttling code](https://github.com/mattdodge/yahoofantasy/blob/abf4dc906b25e4532fbc88de426757c348d492e4/yahoofantasy/cli/dump.py#L56-L73) to try and remedy this a bit in some cases, but it can be annoying.

I'm going to convert this to a discussion though, rather than an issue, as it's not actually a bug with the library
